### DISCUSSION TITLE
using fsl data

### DISCUSSION THREAD

**Discussion by tstefancin** [yeatmanlab/pyAFQ#934] [General]
Hello,
I have data preprocessed using FSL's dti pipeline (DTIFIT). I can't get the data to work in the pipeline. Is there a way to make this data usable with the current parameters or do I need to rerun the data in some other way.

**Comment by arokem** [comment]
Hello @tstefancin : thanks for reporting this! I converted your issue into a discussion so that we can hear a bit more about what you are experiencing. First of all, can you please tell us a bit more about your data? How was it acquired (e.g., spatial resolution, number of directions and b-values)? Second, after running DTIFIT, what did you do next? Did you create a BIDS dataset from which you are working? And how did you run pyAFQ? I believe that you should be able to follow the instructions [here](http://yeatmanlab.github.io/pyAFQ/usage/scalars.html) to use the results of DTIFIT in the pyAFQ pipeline, but I might be missing something. For pyAFQ to work, it will also need access to the processed data through a preproc pipeline derivative (see the linked example).

**Reply by tstefancin** [reply]
My data was acquired as a 2x2x2 voxel size with 64 directions and a b-value of 1000. I created a version of a BIDS directory, but there is no preprocessing pipeline title that matches fsl processing. I am using infant data and I modified the example script for plot_baby_afq.py

**Reply by tstefancin** [reply]
It seems to be a problem with my bvecs file. Do I need to format it another way?
[sub-01_ses-01_dwi.pdf](https://github.com/yeatmanlab/pyAFQ/files/10177343/sub-01_ses-01_dwi.pdf)

**Reply by arokem** [reply]
How do you know there's something wrong with the b-vecs? One thing I noticed is that they are stored in a pdf file. The b-vecs file needs to be a text file.

**Reply by tstefancin** [reply]
When I run the script I get an error that the bids_layout.get_bvec command isn't working and that the list index is out of range

**Reply by arokem** [reply]
Not sure what that means. Could you please share the full stack trace?
